src/index.tsx

```diff
@@ -160,6 +160,32 @@ function loadResources() {
 }
 loadResources();
 
+/**
+ * Set's up Monaco Editor's Workers.
+ */
+enum Workers {
+  Json = 'json',
+  Editor = 'editor',
+}
+
+(window as any).MonacoEnvironment = {
+  getWorkerUrl(moduleId: any, label: string) {
+    if (label === 'json') {
+      return getWorkerFor(Workers.Json);
+    }
+    return getWorkerFor(Workers.Editor);
+  }
```

Comment:
We have all the other workers too, do we need to care about them? The default will be editor.worker.js.
![image](https://github.com/microsoftgraph/microsoft-graph-explorer-v4/assets/8081536/bf4ffc88-374c-4456-aa52-8a8e1945a496)

Reply:
yeah, since they exist I will add them